src/components/table/table.tsx

```diff
@@ -87,6 +88,15 @@ export class Table
    */
   @Event() swipeLeft: EventEmitter;
 
+  @Listen("click", { capture: true })
+  handleClick(event: UIEvent) {
+    event.preventDefault();
```

Comment:
Why are we preventing the default behavior here?

Reply:
Same to previous logic, prevent possible default functions. I will remove that.

Reply:
I'm accepting this PR, but please don't forget to remove the `preventDefault` call

Reply:
I will commit this last change and merge the PR